libs/components/code-examples/src/lib/modules/lists/list-summary/basic/example.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component } from '@angular/core';
+import { SkyListSummaryModule } from '@skyux/lists';
+
+/**
+ * @title List summary with basic setup
+ */
+@Component({
+  selector: 'app-lists-list-summary-basic-example',
+  templateUrl: './example.component.html',
+  imports: [SkyListSummaryModule],
+})
+export class ListsListSummaryBasicExampleComponent {
+  protected summaryItems = [
+    {
+      label: 'Total records',
+      value: 1247,
+      helpPopoverContent: 'The total number of records in the current dataset.',
+      helpPopoverTitle: 'Total Records Help',
```

Comment:
```suggestion
      helpPopoverTitle: 'Total records help',
```
We probably don't need a title here, but no worries if it's valuable to include an example.